table/column_test.go

```diff
@@ -246,64 +250,70 @@ func (t *testTableSuite) TestGetZeroValue(c *C) {
 	}
 	sc := new(stmtctx.StatementContext)
 	for _, tt := range tests {
-		colInfo := &model.ColumnInfo{FieldType: *tt.ft}
-		zv := GetZeroValue(colInfo)
-		c.Assert(zv.Kind(), Equals, tt.value.Kind())
-		cmp, err := zv.CompareDatum(sc, &tt.value)
-		c.Assert(err, IsNil)
-		c.Assert(cmp, Equals, 0)
+		t.Run(fmt.Sprintf("%+v \n", tt.ft), func(t *testing.T) {
```

Comment:
```suggestion
		t.Run(fmt.Sprintf("%+v", tt.ft), func(t *testing.T) {
```

can we reduce extra bytes?

Reply:
i want to make the test logs more clean, i assume an `new line` could be better but need confirm.
will check later.

Reply:
fixed